easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.53.5'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['991421f41a4ed4cc1637e5f9db0d03fd236d2cbd19f3c5b097a343bec5126602']
+
+builddependencies = [
+    ('binutils', '2.27'),
+    ('Python', '2.7.13', '-bare'),
+]
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.8.1'),
+    ('libxml2', '2.9.4'),
+    ('PCRE', '8.41'),
+    ('util-linux', '2.30.1'),
+]
+
+# avoid using hardcoded path to Python binary in build step
+preconfigopts = "export PYTHON=python && "
+
+configopts = "--disable-maintainer-mode --disable-silent-rules --disable-libelf --disable-systemtap "
+configopts += "--enable-static --enable-shared"
+
+postinstallcmds = ["sed -i -e 's|#!.*python|#!/usr/bin/env python|' %(installdir)s/bin/*"]
+
+sanity_check_paths = {
+    'files': ['lib/libglib-%(version_major)s.0.a', 'lib/libglib-%%(version_major)s.0.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include'],
+}
```

Comment:
```python
modextrapaths = {'CPATH': ['include/glib-2.0', 'lib/glib-2.0/include']}
```

Hmm, this is only present in one single (fairly old) `GLib` easyconfig (`GLib-2.44.0-GCC-4.9.2.eb`).

It seems like this is not needed at all in the others, which may mean it's not required there either...

Should be easy to tell, since it's only used as a dep for `Qt-4.8.6-foss-2015a-GLib-2.44.0.eb` (which in turns is a dep for `CGAL-4.6-*-2015a-Python-2.7.*.eb`), nowhere else...